python/docs/conf.py

```diff
@@ -6,6 +6,8 @@
 import os
 import sys
 sys.path.insert(0, os.path.abspath('../python'))
+if sys.platform == 'win32':
+    sys.path.insert(0, os.path.abspath('../python/x64/Release'))
```

Comment:
This was necessary to get the doc-build (`make html` with `sphinx`) to work on Windows.
Because Windows outputs the `IcePy` module into these nested folders you know.